### PR TITLE
Edits on first markdown version of draft

### PR DESCRIPTION
**Pull request by ptsefton**
- Adds a new type to @context WorkflowSketch (ro-terms:Sketch).
-  Added Pronom URIs for sketch example as per spec (but maybe we SHOULD support mime types)
-  Cleaned up a few inconsistencies with SHOULD and MAY
-  Made including creator as a Context Entity a MUST

TODO: s/creator/author/g

### CODE REVIEW

**Comment by stain** [comment]
You changed the syntax to `[Journal] is mapped to <http://schema.org/Periodical>.` - but I am not sure what the syntax with `[ ]` is meant to do as there is no hyperlink. Does not the ` syntax work better?

**Comment by ptsefton** [comment]
Just need to define it at the end of the file.

File: docs/0.2-DRAFT/index.md [hunk 1]
```diff
@@ -173,9 +172,25 @@ RO-Crate relies heavily on [schema.org](http://schema.org) using a constrained s
 Generally, the standard keys for [schema.org](http://schema.org) should be used. However, RO-Crate uses variant names for some elements, specifically:
 
 
-*   `File` is mapped to <http://schema.org/MediaObject> which was chosen as a compromise as it has many of the properties that are needed to describe a generic file. Future versions of schema.org or a research data extension may re-define `File`.
-*   `Journal` is mapped to <http://schema.org/Periodical>.
-*   `path` is mapped to <http://schema.org/contentUrl>. This property is used on some classes which do not strictly (yet) allow it in the official Schema.org.
+*   [File] is mapped to <http://schema.org/MediaObject> which was chosen as a compromise as it has many of the properties that are needed to describe a generic file. Future versions of schema.org or a research data extension may re-define `File`.
+*   [Journal] is mapped to <http://schema.org/Periodical>.
+*   [path] is mapped to <http://schema.org/contentUrl>. This property is used on some classes which do not strictly (yet) allow it in the official Schema.org.
+
```
Comment (eocarragain) [inline docs/0.2-DRAFT/index.md]: As discussed - as this is the first time these are being mentioned, they shouldn't be hyperlinked to the schema.org description.

File: docs/0.2-DRAFT/index.md [hunk 2]
```diff
@@ -173,9 +172,25 @@ RO-Crate relies heavily on [schema.org](http://schema.org) using a constrained s
...
+`WorkflowSketch` - for an image depicting a workflow is from the ro-terms ontology <http://purl.org/ro/wf4ever#Sketch>, see the [documentation](http://wf4ever.github.io/ro/2016-01-28/roterms/#Sketch.)
+
+ RO-Crate also uses the _Portland Common Data Model_ ([PCDM](https://github.com/duraspace/pcdm/wiki)) and imports these terms:
+ 
+ [RepositoryObject](https://pcdm.org/2016/04/18/models#Object), 
+ [RepositoryCollection](https://pcdm.org/2016/04/18/models#Collection)
+ [RepositoryFile](https://pcdm.org/2016/04/18/models#Collection)
+ [hasMember](https://pcdm.org/2016/04/18/models#hasMember)
+ [hasFile](https://pcdm.org/2016/04/18/models#hasFile)
+
```
Comment (eocarragain) [inline docs/0.2-DRAFT/index.md]: as above, the aliases shouldn't be hyperlinked here and the format should be the same as above, i.e.
`RepositoryObject` is mapped to [https://pcdm.org/2016/04/18/models#Object](https://pcdm.org/2016/04/18/models#Object),

File: docs/0.2-DRAFT/index.md [hunk 3]
```diff
@@ -173,9 +172,25 @@ RO-Crate relies heavily on [schema.org](http://schema.org) using a constrained s
...
+## Additional metadata standards
+
+`WorkflowSketch` - for an image depicting a workflow is from the ro-terms ontology <http://purl.org/ro/wf4ever#Sketch>, see the [documentation](http://wf4ever.github.io/ro/2016-01-28/roterms/#Sketch.)
+
+ RO-Crate also uses the _Portland Common Data Model_ ([PCDM](https://github.com/duraspace/pcdm/wiki)) and imports these terms:
+ 
+ [RepositoryObject](https://pcdm.org/2016/04/18/models#Object), 
+ [RepositoryCollection](https://pcdm.org/2016/04/18/models#Collection)
+ [RepositoryFile](https://pcdm.org/2016/04/18/models#Collection)
```
Comment (eocarragain) [inline docs/0.2-DRAFT/index.md]: should be https://pcdm.org/2016/04/18/models#File

File: docs/0.2-DRAFT/index.md [hunk 4]
```diff
@@ -323,13 +336,15 @@ The following _RO-Crate Metadata file _represents a minimal description of an_ R
 
 ### Examples of referencing Data Entities (files and folders) from the Root Data Entity
 
-Where files and folders are represented as _Data Entities_ in the RO-Crate JSON-LD, these MUST be linked to from the Root Data Entity using the hasPart property. 
+Where files and folders are represented as _Data Entities_ in the RO-Crate JSON-LD, these MUST be linked to from the Root Data Entity using the [hasPart] property. 
 
-Where files are contained in sub-folders of the RO-Crate root, these MUST NOT be referenced from the Root Data Entity; rather, they should be linked from their parent Data Entity. 
+[hasPart](http://schema.org/hasPart)
```
Comment (eocarragain) [inline docs/0.2-DRAFT/index.md]: remove floating hasPart link

File: docs/0.2-DRAFT/index.md [hunk 5]
```diff
@@ -1394,7 +1408,7 @@ If [thumbnail](http://schema.org/thumbnail)s are incidental to the data set, the
 
 (These are notes for now)
 
-DataCite can accept metadata in Schema.org format if it is framed correctly via the Bolognese crosswalk to. 
+DataCite can accept metadata in Schema.org format if it is framed correctly, via the Bolognese crosswalk.
```
Comment (eocarragain) [inline docs/0.2-DRAFT/index.md]: Discussed this & decided to remove DataCite/Bolognese reference in favour of just saying that, when displaying an RO-Crate website, a textual citation should be displayed (possibly with reference to [Force11 Data Citation Principles](https://www.force11.org/datacitationprinciples))

File: docs/0.2-DRAFT/index.md [hunk 6]
```diff
@@ -173,9 +172,25 @@ RO-Crate relies heavily on [schema.org](http://schema.org) using a constrained s
...
+*   [File] is mapped to <http://schema.org/MediaObject> which was chosen as a compromise as it has many of the properties that are needed to describe a generic file. Future versions of schema.org or a research data extension may re-define `File`.
+*   [Journal] is mapped to <http://schema.org/Periodical>.
+*   [path] is mapped to <http://schema.org/contentUrl>. This property is used on some classes which do not strictly (yet) allow it in the official Schema.org.
+
+
+## Additional metadata standards
+
+`WorkflowSketch` - for an image depicting a workflow is from the ro-terms ontology <http://purl.org/ro/wf4ever#Sketch>, see the [documentation](http://wf4ever.github.io/ro/2016-01-28/roterms/#Sketch.)
+
+ RO-Crate also uses the _Portland Common Data Model_ ([PCDM](https://github.com/duraspace/pcdm/wiki)) and imports these terms:
```
Comment (eocarragain) [inline docs/0.2-DRAFT/index.md]: Do we need to say more about why these PCDM properties have been included? 

In particular, I think it is important to say when to use hasPart vs hasMember/hasFile and when to use RepositoryFile vs File/MediaObject

**Pull request merged**